hosted/eks/p0/p0_suite_test.go

```diff
@@ -66,3 +71,84 @@ var _ = ReportAfterEach(func(report SpecReport) {
 	// Add result in Qase if asked
 	Qase(testCaseID, report)
 })
+
+func p0upgradeK8sVersionChecks(cluster *management.Cluster, client *rancher.Client, clusterName string) {
+	helpers.ClusterIsReadyChecks(cluster, client, clusterName)
+
+	versions, err := helper.ListEKSAvailableVersions(client, cluster.ID)
+	Expect(err).To(BeNil())
+	Expect(versions).ToNot(BeEmpty())
+	upgradeToVersion := &versions[0]
+	GinkgoLogr.Info(fmt.Sprintf("Upgrading cluster to EKS version %s", *upgradeToVersion))
+
+	By("upgrading the ControlPlane", func() {
+		var err error
+		cluster, err = helper.UpgradeClusterKubernetesVersion(cluster, upgradeToVersion, client)
+		Expect(err).To(BeNil())
+		err = clusters.WaitClusterToBeUpgraded(client, cluster.ID)
+		Expect(err).To(BeNil())
+		Expect(cluster.EKSConfig.KubernetesVersion).To(BeEquivalentTo(upgradeToVersion))
+	})
+
+	// Does not upgrades version since using custom LT, skip for imported cluster
+	Expect(helpers.TestConfig).ToNot(BeEmpty())
+	if strings.Contains(helpers.TestConfig, "provisioning") {
```

Comment:
In the next round, I was thinking to add a new field to helpers.Context struct called `TestingImport` `=string.Contains(helpers.TestConfig, "import")` and use it so that we have a common source of truth, but this is good enough for now, thank you for implementing this. :raised_hands: